platform-jobs/samza/post-publish-processor/src/main/java/org/ekstep/job/samza/service/PostPublishProcessor.java

```diff
@@ -134,9 +134,10 @@ public void processMessage(Map<String, Object> message, JobMetrics metrics, Mess
                 String nodeId = (String) object.get("id");
                 String name = (String) edata.getOrDefault("name", "");
                 String createdBy = (String) edata.getOrDefault("createdBy", "");
+                List<String> createdFor = (List<String>) edata.getOrDefault("createdFor", "");
```

Comment:
List<String> createdFor = (List<String>) edata.getOrDefault("createdFor", "");

to 
List<String> createdFor = (List<String>) edata.getOrDefault("createdFor", new ArrayList<String>());

Reply:
Fixed it .